tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "module": "es2022",
```

Comment:
Does this also need to be addressed for node 14? es2022 allows top level await, which was added in [Node v14.8.0](https://nodejs.org/en/blog/release/v14.8.0/#:~:text=Derek%20Lewis)%20%2334602-,%5B62bb2e757f%5D,-%2D%20(SEMVER%2DMINOR)%20module)

Reply:
Our min version is currently actually `14.14.0`, so we can use top-level await.

I'm hoping people who use npm 14 at least use a _recent_ npm 14 -- if they don't then we'll have to also drop top-level await, `fs.rm`, and anything else that wasn't in `node 14.0.0`.